source/administration/changelog.md

```diff
@@ -4,6 +4,105 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+
+#### Added multi-factor authentication support to Team Edition
+ - See more details on [this Forum post](https://forum.mattermost.org/t/multi-factor-authentication-mfa-in-team-edition/6287).
+ 
+#### Enhanced image performance
+ - Improved performance for images by adding support for image proxy servers, which are now integrated into the server and switched on by default.
+
+### Improvements
+
+#### User Interface (UI)
+ - Improved sorting of emoji in the emoji autocomplete and emoji picker search results.
+ - Added support for emoji picker for mobile web view.
+
+#### Notifications
+ - Added a channel notification setting to disable at-channel mentions.
+ 
+#### Administration
+ - Added the ability to search users by role in **System Console** > **Users**.
+ - Added a CLI command to modify an outgoing webhook.
+ - Added a CLI command to restore a team.
+
+#### Performance
+ - Made posting idempotent.
+
+#### Slash Commands
+ - Added support for sending a message to a different channel than where the slash command was issued from.
+ - Improved error handling of slash commands.
+ - Added an option to send a message when an invalid slash command is entered on the right-hand side.
+
+#### Plugins
+ - Added server support for updating a plugin instead of having to remove and install them as two separate actions.
+ 
+#### Attachments
+ - Optimized file attachment memory usage where possible.
+ 
+#### Integrations 
+ - Enabled "true"/"false" for bool values in webhook payload.
+
+### Bug Fixes
+ 
+ - Fixed an issue where "[user] is typing ..." was not removed when a message was composed and sent very quickly.
+ - Fixed an issue where an announcement banner displayed when the banner was enabled but the text field was blank.
+ - Fixed an issue where a language was not set if selected in Account Settings.
+ - Fixed an issue where removing rows from Send Email Invite modal didn't remove them immediately.
+ 
+### Compatibility
+
+### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+ 
+#### Changes to Team Edition and Enterprise Edition:
+ 
+ - Under ``"ServiceSettings"`` in ``config.json``:
+    - Added ``"ExperimentalLdapGroupSync": false``, ``"GroupFilter": ""``, ``"GroupDisplayNameAttribute": ""``, ``"GroupIdAttribute": ""`` and ``"ExperimentalStrictCSRFEnforcement": false``, to add the ability to configure group display name and unique identifier.
+ - Under ``"ImageProxySettings":`` in ``config.json``:
+    - Added ``"Enable": true,``, ``"ImageProxyType": "local",``, ``"RemoteImageProxyURL": "",`` and ``"RemoteImageProxyOptions": ""``, to allow integrating image proxy into the server and switching it on by default.
+ - Under ``"ExperimentalSettings":`` in ``config.json``:
+    - Added ``"LinkMetadataTimeoutMilliseconds": 5000``, to
```

Comment:
What about the settings for experimental post metadata?

"DisablePostMetadata": false, was added under experimental settings